.github/ISSUE_TEMPLATE/04_core_feature_request.yml

```diff
@@ -0,0 +1,62 @@
+name: Core Feature Request
+description: Create a feature request for Azure.Core
+title: "[Core] "
+labels: ["Azure.Core", "feature-request"]
+assignees: ["christothes","annelo-msft", "Josh-Love-msft"]
+
+body:
+  - type: dropdown
+    id: apichange
+    attributes:
+      label: API Change Needed
+      description: Is a public API change needed for this feature?
+      options:
+        - "Yes"
+        - "No"
+        - "Unsure"
+    validations:
+      required: true
+  - type: textarea
+    id: background
+    attributes:
+      label: Background and motivation
+      description: Please describe the purpose and value of the new feature and/or API here.
+      placeholder: Purpose
+    validations:
+      required: true
+  - type: textarea
+    id: api-proposal
+    attributes:
+      label: API / Feature Proposal
+      description: |
+        Please provide the specific public API signature diff that you are proposing and/or proposed feature designs
```

Comment:
Could we suggest using APIView for this as an option?

Reply:
The only downside to that is that it requires that the changes be stubbed out, and packaged in order to get them there. The changes may only be conceptual at this phase. But I think it is worth mentioning APIView.